data/src/main/java/com/jjswigut/data/models/Group.kt

```diff
@@ -0,0 +1,7 @@
+package com.jjswigut.data.models
+
+data class Group(
```

Comment:
how come all of these are nullable? does it make sense to have a group if everything is null? can we enforce nullability higher up in the stream?